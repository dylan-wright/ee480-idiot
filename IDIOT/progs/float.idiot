addf $u0, $u0
f2i $u0, $u0
i2f $u0, $u0
invf $u0, $u0
mulf $u0, $u0
