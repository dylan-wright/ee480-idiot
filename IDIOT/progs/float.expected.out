Text:
//generated by AIK version 20070512
@0000
8000
8000
8000
8000
8000
//end
Data:
//generated by AIK version 20070512
//end
